recipe/meta.yaml

```
@@ -1,31 +1,22 @@
 {% set name = "cupy" %}
-{% set version = "8.0.0b2" %}
-{% set sha256 = "6e06ccd1795b1c6f7fd82ff6f5db8306c69ca39393976ca49e9bc687646f9dd0" %}
+{% set version = "8.0.0rc1" %}
+{% set sha256 = "2981c809f7a7e1964c83921bcc065ae573e019f48f64468dc345bb3cd451e84e" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  - url: https://github.com/cupy/cupy/archive/v{{ version }}.tar.gz
+  - url: https://pypi.io/packages/source/c/cupy/cupy-{{ version }}.tar.gz
```

We like parametrization of such cases :)
```suggestion
  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

---

Thanks!